projects/addon-editor/components/toolbar-tools/table-size-selector/table-size-selector.component.ts

```diff
@@ -20,7 +20,15 @@ const MIN_DISTANCE_PX = 70;
 })
 export class TuiTableSizeSelectorComponent {
     @Output()
-    readonly onSelectSize = new EventEmitter<{cols: number; rows: number}>();
+    readonly selectSize = new EventEmitter<{cols: number; rows: number}>();
+
+    /**
+     * @deprecated use {@link selectSize}
+     * TODO: remove in v4.0
+     */
+    @Output()
+    // eslint-disable-next-line @angular-eslint/no-output-on-prefix
+    readonly onSelectSize = this.selectSize;
```

Comment:
Should also update `onClick` method in this component.

Reply:
fixed